app-center-services/src/main/resources/locale/addon/appcenter_en.properties

```diff
@@ -18,13 +18,23 @@ appCenter.editpref.form.pageSize=Max applications
 appCenter.editpref.form.save=Save
 search.connector.label.application=Applications
 
+appCenter.system.application.agenda=Agenda
+appCenter.system.application.drives=Drives
+appCenter.system.application.forum=Forum
+appCenter.system.application.news=News
+appCenter.system.application.notes=Notes
+appCenter.system.application.perkStore=Perk store
+appCenter.system.application.tasks=Tasks
+appCenter.system.application.wallet=Wallet
+appCenter.system.application.wiki=Wiki
+
 appCenter.adminSetupForm.generalSettings=Settings
 appCenter.adminSetupForm.maxFavoriteApps=Number of favorites allowed
 appCenter.adminSetupForm.defaultAppImage=Default app icon
 appCenter.adminSetupForm.applications=Applications
 appCenter.adminSetupForm.addNewApp=Add an application
 appCenter.adminSetupForm.editApp=Edit application
-appCenter.adminSetupForm.DeleteApp=Remove an application
+appCenter.adminSetupForm.modal.DeleteApp=Are you sure ?
```

Comment:
Did you change on crowdin ?

Reply:
Not on crowdin but I updated the label name and value.

Reply:
:+1: